### PR TITLE
raising profile photo size to comply with frio sidebar profile photo scaling

### PR DESCRIPTION
**Pull request by vinzv**
fixing #5992

### CODE REVIEW

**Comment by vinzv** [comment]
I changed the default avatar and added a 300px version of it, if that's what you meant.

For addons I don't think there are changes needed as my initial commit is only for the automated resizing of uploaded profile pictures.

File: mod/dfrn_confirm.php [hunk 1]
```diff
@@ -537,7 +537,7 @@ function dfrn_confirm_post(App $a, $handsfree = null)
 		if (DBA::isResult($contact)) {
 			$photo = $contact['photo'];
 		} else {
-			$photo = System::baseUrl() . '/images/person-175.jpg';
+			$photo = System::baseUrl() . '/images/person-300.jpg';
```
Comment (annando) [inline mod/dfrn_confirm.php]: There is no "person-300.jpg" on our system.

**Comment by annando** [comment]
We have several image sizes for the contact. I'm unsure which size the biggest one does have. Possibly this already helps.

File: mod/dfrn_confirm.php (hunk 1 above)
Comment (MrPetovan) [reply mod/dfrn_confirm.php]: The PR actually creates one.

**Comment by MrPetovan** [comment]
@annando Please refer to the image scale list I posted earlier.

File: mod/dfrn_confirm.php (hunk 1 above)
Comment (vinzv) [reply mod/dfrn_confirm.php]: I added it.

**Comment by vinzv** [comment]
So I went to the whole code and changed 175 to 300 where it seemed appropriate.

File: mod/profile_photo.php [hunk 2]
```diff
@@ -288,7 +288,7 @@ function profile_photo_crop_ui_head(App $a, Image $image)
 	$height = $image->getHeight();
 
 	if ($width < 175 || $height < 175) {
```
Comment (MrPetovan) [inline mod/profile_photo.php]: I just realized that those values probably need to be raised to 300 as well.

**Comment by vinzv** [comment]
Yeah, I didn't touch the themes to avoid any breakage. Browser's downscaling should be fine.

File: mod/profile_photo.php (hunk 2 above)
Comment (vinzv) [reply mod/profile_photo.php]: Don't think so, as this is for **up**scaling.

Comment (MrPetovan) [reply mod/profile_photo.php]: You are probably right, let's see how this pans out.

**Pull request merged**